src/deskstar-backend/Deskstar/Controllers/AuthController.cs

```diff
@@ -37,9 +37,10 @@ public IActionResult CreateToken(CreateTokenUser user)
     [AllowAnonymous]
     public IActionResult Register(RegisterUser registerUser)
     {
-        if (!_authUsecases.registerUser(registerUser))
+        var result = _authUsecases.RegisterUser(registerUser);
+        if (result != RegisterReturn.Ok)
         {
-            return BadRequest();
+            return BadRequest(result);
```

Comment:
Does it make sense to return that here? Isn't that just a number?.. Maybe we should discuss on wether we want to return a more descriptional text status code.

Reply:
no :D